html/components/_buttons.scss

```diff
@@ -179,7 +166,7 @@ Removes extra padding and border in Firefox when using `input type="submit"`
 .sprk-c-Button--tertiary.sprk-is-Disabled,
 .sprk-c-Button--tertiary[disabled] {
   box-shadow: $sprk-btn-tertiary-disabled-shadow;
-  border: $sprk-btn-tertiary-disabled-border;
+  border-color: $sprk-btn-tertiary-disabled-border-color;
```

Comment:
Using border vs. border-color: Using border gives us more room to change things via the variable. Scoping this to just border-color makes it so we can't / app teams can't easily change the other properties of the border.